examples/publish-wav/index.ts

```diff
@@ -41,9 +41,10 @@ const dataSize = sample.readUInt32LE(40) / 2;
 // set up audio track
 const source = new AudioSource(sampleRate, channels);
 const track = LocalAudioTrack.createAudioTrack('audio', source);
-const options = new TrackPublishOptions();
+const options = {
+  source: TrackSource.SOURCE_MICROPHONE,
+} as TrackPublishOptions;
 const buffer = new Int16Array(sample.buffer);
-options.source = TrackSource.SOURCE_MICROPHONE;
```

Comment:
@lukasIO I was a little surprised this was broken and wonder if it means that we aren't exporting the API compatibility helpers you added in the way you expected? I didn't dig in but it might be worth a quick look

Reply:
The helpers are exposed, I forgot to initialise the contructor arguments with an empty object to allow for passing no params at all 🤦 , sorry! 

Fixed in #310